ledger-core/v2/conveyor/smachine/state_update_type.go

```diff
@@ -186,10 +186,10 @@ func (v StateUpdateType) verifyVar(u interface{}) interface{} {
 }
 
 func (v StateUpdateType) ShortLoop(slot *Slot, stateUpdate StateUpdate, loopCount uint32) bool {
-	if v.shortLoop == nil {
-		return false
-	}
-	return v.shortLoop(slot, stateUpdate, loopCount, v.stepDeclaration)
+	// if v.shortLoop == nil {
```

Comment:
Are you sure?

Reply:
@cyraxred Can you confirm this part?

Reply:
In any case, I think it should be in separate PR

Reply:
split it in different branch and create PR https://github.com/insolar/assured-ledger/pull/113